src/app/course/challenge/course-challenge-create-edit/course-challenge-create-edit.component.html

```diff
@@ -1,4 +1,13 @@
 <div class="tui-container tui-container_adaptive">
+    <a
+        [routerLink]="eventId ? ['../..'] : ['..']"
+        icon="tuiIconArrowLeft"
+        size="s"
+        tuiButton
+        appearance="flat"
+    >
+        Back to List of Challenges
+    </a>
     <form (ngSubmit)="onSubmit()" [formGroup]="challengeForm">
         <h2 class="tui-form__header">{{eventId ? 'Edit ' + event?.name : 'Create New Challenge'}}</h2>
```

Comment:
Now that you are adding this you can also fix the header to be consistent on the website. (You can use the headers in goal pages as reference).

Also if you don't want to do this because it's not in the class, that's fine just ignore this comment.

Reply:
I'm not quite sure which header you are referring to, could you attach a screenshot or elaborate?

Reply:
![image](https://user-images.githubusercontent.com/22093299/213813476-99bca2ca-9752-4488-9064-c7374847f726.png)
The picture above is in the goal pages. The title sizes and the margins are highlighted.

in the goals, we use an `h5` tag with the class name `tui-text_h5`. The margin between the header and the navigation button above is set by `mt-3`.

```
<div class="flex flex-wrap gap-3 mt-3">
        <h5 class="tui-text_h5 tui-col_stretch flex-grow">
            Active Goals
        </h5>
        <a class="min-w-200 mr-4" tuiButton size="m" routerLink="create">Create a new goal</a>
    </div>
```

Here an `h2` tag is used with the `tui-form__header` class. The margin is also set by the class itself.

```
<h2 class="tui-form__header">{{eventId ? 'Edit ' + event?.name : 'Create New Challenge'}}</h2>
```

Now we want to have them consistent on all the pages.